### PR TITLE
Add synonyms to Neurofibromatosis 2

### LINKED ISSUE #7834: Neurofibromatosis 2
Hi!  This is a request to add the following synonyms to Neurofibromatosis 2:

- NF2-Related Schwannomatosis
- Full NF2
- Full neurofibromatosis type 2
- Nonmosaic NF2-related schwannomatosis
- Nonmosaic neurofibromatosis type 2

Sources: 
https://www.ncbi.nlm.nih.gov/books/NBK1201/ 
https://www.orpha.net/en/disease/detail/637

Note that we have a NORD report entitled NF2-Related Schwannomatosis

Thank you!  Gioconda

### PR DESCRIPTION
**Pull request by matentzn**
close #7834

@nicolevasilevsky please review this PR, thanks!

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -139116,6 +139116,8 @@ synonym: "acoustic schwannomas bilateral" RELATED [GARD:0007193]
 synonym: "acoustic Schwannomas, bilateral" RELATED [OMIM:101000]
 synonym: "bilateral acoustic neurofibromatosis" EXACT [NCIT:C3274, OMIM:101000, Orphanet:637]
 synonym: "central neurofibromatosis" EXACT [NCIT:C3274, Orphanet:637]
+synonym: "full neurofibromatosis type 2" EXACT [Orphanet:637]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:139119]: Please add the dbxef that was requested on the ticket: https://www.ncbi.nlm.nih.gov/books/NBK1201/

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -426792,8 +426798,7 @@ subset: orphanet_rare {source="Orphanet:636"}
 subset: otar {source="MONDO:OTAR"}
 subset: rare
 synonym: "neurofibromatosis" BROAD [NCIT:C3273]
-synonym: "Neurofibromatosis 1" EXACT [NORD:1502]
-synonym: "neurofibromatosis 1" EXACT [NCIT:C3273]
+synonym: "neurofibromatosis 1" EXACT [NCIT:C3273, NORD:1502]
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.obo:426801]: these changes were not requested on the ticket and are not mentioned in the commit message. Could you please explain these additional changes?
Thanks!

**Review by nicolevasilevsky — CHANGES_REQUESTED** [review]
See comment inline, minor changes needed. Thanks for adding these synonyms and addressing this ticket.

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:139119]: done

**Comment by nicolevasilevsky** [comment]
@twhetzel can you please review this PR?

Note, I also added some synonyms to type 1, for teaching purposes at the ICBO tutorial

**Review by twhetzel — APPROVED** [review]

**Comment by twhetzel** [comment]
@nicolevasilevsky since you formally requested changes in this PR, you'll need to approve also.

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**